README.md

```diff
@@ -28,3 +28,11 @@ The tests implemented so far are integration (not unit) tests, so a working WAR
 3. Run `mvn jetty:run` from the eva-server subfolder
 4. Run `mvn test` from the root folder
 
+Enabling Oauth2 Security
+------------------------
+
+In order to enable Oauth2 Security you must enable the `oauth2-security` profile in the spring-boot application in any
+way supported by Spring-boot as stated in [Spring documentation](http://docs.spring.io/spring-boot/docs/current/reference/html/howto-properties-and-configuration.html#howto-set-active-spring-profiles). In adition, the application requires configuration to point to the public user check endpoints
+of your oauth2 compatible authentication service. More information is available in [Spring documentation](http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-security.html#boot-features-security-oauth2-resource-server).
+
+When using Oauth2 Security, swagger UI will be still enabled and public to review the API but no command sent from the ui will be authorized.
```

Comment:
But then I've found [this](http://www.appsdev.is.ed.ac.uk/blog/?p=820)... I'm not sure of how it will go. I need to test fully the example.